src/popup/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -241,10 +241,9 @@ export default {
     if (this.transaction.contractId) {
       try {
         this.loading = true;
-        const { getSdk } = useSdk({ store: this.$store });
```

Comment:
@CedrikNikita This throws an error now. Store not found.

Reply:
Behaviour o this composable had changed during our work on this PR, and on rebase we were suppose to change it as well.